src/current/_includes/releases/v24.1/v24.1.0-alpha.1.md

```diff
@@ -140,106 +138,57 @@ Release Date: March 7, 2024
-- Fixed a bug that caused the sql.txns.open.internal and sql.statements.active.internal metrics to never get updated. These metrics are gauges that count the number of transactions and statements being executed by operations internal to CockroachDB; they do not include any operations executed by the external SQL client. This bug was present since v22.1.0. [#119251][#119251]
-- Crdb_internal.leases could cause a node to become unavailable due to a deadlock in the leasing subsystem. [#119305][#119305]
-- If an individual replica's circuit breaker had tripped but the range was otherwise functional, e.g. because the replica was partially partitioned away from the leaseholder, it was possible for a gateway to persistently error when contacting this replica instead of retrying against a functional leaseholder elsewhere. The gateway will now retry such errors against other replicas once. [#118737][#118737]
-- Fixes a bug in the webhook sink where the http request body may not be initialized on retries, resulting in the error "http: ContentLength=... with Body length 0". [#119326][#119326]
-- Fixed a bug where rangefeed resolved timestamps could get stuck, continually emitting the log message "pushing old intents failed: range barrier failed, range split", typically following a range merge. [#119512][#119512]
+- Fixed a rare panic that could happen during a pg_dump import that contains a function that has a subquery in one of its arguments, like `SELECT addgeometrycolumn(...)`. Now, attempting to import a pg_dump with such a function results in an expected error. [#118569][#118569]
+- [AUTO CREATE STATS]({% link v23.2/show-jobs.md}#show-automatic-jobs) jobs could previously lead to growth in an internal system table resulting in slower job-system related queries. [#118589][#118589]
+- Fixed an issue in CockroachDB where, if operating on a Linux system outside of a CPU cgroup, the system would repeatedly log the error `unable to get CPU capacity` at 10-second intervals [#118657][#118657]
+- Fixed a bug where casts of floats to integers simply truncated the decimal portion. These casts now match the PostgreSQL behavior of rounding to the nearest integer, and in cases of a value halfway between two integers, rounding to the nearest even number. This aligns with the "round half to even" rule or "bankers' rounding", offering greater overall precision across a group of such cast operations. [#117798][#117798]
+- Fixed a bug where statements like `ADD COLUMN j INT, ADD UNIQUE WITHOUT INDEX (j)`, which add new columns with unique constraints without creating associated indexes, would fail with an internal error. [#118291][#118291]
+- Previously, altering from a [Regional By Row]({% link v23.2/alter-table.md}#regional-by-row) table to a Regional By Table table could cause leaseholders to never move to the databse's primary region. This is now fixed. [#118001][#118001]
```

Comment:
Link Regional By Table to `alter-table.html#regional-by-table`.

Typo on "database".